File: Course_01/C Language/Vetores/media_pares.c
#include <stdio.h>
#include <stdlib.h>

int main()
{
    int n;

    printf("Quantos elementos vai ter o vetor? ");
    scanf("%d", &n);

    int a[n], countPares, somaPares;
    double mediaPares;

    for (int i = 0; i < n; i++) {
        printf("Digite um numero: ");
        scanf("%d", &a[i]);
    }
    countPares = 0;
    somaPares = 0;

    for (int i = 0; i < n; i++) {
        if (a[i] % 2 == 0) {
            countPares++;
            somaPares+=a[i];
        }
    }

    if (countPares == 0) {
        printf("NENHUM NUMERO PAR");
    }
    else {
        mediaPares = (double) somaPares / countPares;
        printf("MEDIA DOS PARES = %.1lf", mediaPares);
    }



    return 0;
}

File: Course_01/Java Language/Vetores/media_pares.java
package Vetores;

import java.util.Locale;
import java.util.Scanner;

public class media_pares {

	public static void main(String[] args) {
		Locale.setDefault(Locale.US);
		Scanner sc = new Scanner(System.in);
		
		int i, n, contePar;
		
		System.out.print("Quantos elementos vai ter o vetor? ");
		n = sc.nextInt();
		
		double [] v = new double [n];
		double soma, media;
		
		for (i = 0; i < n; i++) {
			System.out.print("Digite um numero: ");
			v[i] = sc.nextDouble();
		}
		soma = 0;
		contePar = 0;
		
		for (i = 0; i < n; i++) {
			if (v[i] % 2 == 0) {
				soma += v[i];
				contePar++;
			}
		}
		if (contePar != 0) {
			media = soma / contePar;
			System.out.println("MEDIA DOS PARES = " + String.format("%.1f", media));
		}
		else {
			System.out.println("NENHUM NUMERO PAR");
		}
		
		sc.close();
	}

}
